### ISSUE TITLE
Updating the dependencyGraph.ts toDotFormat() method to use more attributes

### ISSUE THREAD

**Issue by FishOfPrey** [afawcett/dependencies-cli#19]
The current Graphviz .dot output has hardcoded HTML in the label. This is fine if you then feed the file directly into the GraphViz CommandLine, but is limiting for other software that can work is same file format.

I'd like to suggest either changing the toDotFormat() method or having and optional input that defines the output format. 

For instance, I changed:

`dot += `  X${node.name} [label=<${node.details.get('parent')}${node.details.get('name')}<BR/><FONT POINT-SIZE="8">${node.details.get('type')}</FONT>>]\n`;`

to

`dot += `  X${node.name} [label=<${node.details.get('parent')}${node.details.get('name')}}>, type=<${node.details.get('type')}>]\n`;`

This allows me to partition the nodes in the output graph by type and doesn't force the type information into the nodes label.

![image](https://user-images.githubusercontent.com/2396092/43934614-43603858-9ca4-11e8-98f8-59a362d4e5cb.png)

E.g. See how some custom fields are clustered around certain layouts.

![image](https://user-images.githubusercontent.com/2396092/43934658-7c578e2c-9ca4-11e8-9c59-a169a1977db4.png)

**Comment by FishOfPrey** [comment]
@afawcett I'm using [Gephi](https://gephi.org/). It's very powerful for graph visualization, but I ran into some limitations with it's support for the dot format. The image above was using their dot format import.

In the end I made my own graph representation from the dependency API and exported it into their native format. The full details of what I ended up with are in [Dreamforce 2018 Presentation - Understand your Org shape via visualization of Metadata Component Dependencies](http://www.fishofprey.com/2018/09/happysoupmix.html)
